docs/README.md

```diff
@@ -0,0 +1,34 @@
+# Metasploit docs site
+
+## Setup 
+
+### Developer build
+
+Builds the latest docs content from the existing [Metasploit Wiki](https://github.com/rapid7/metasploit-framework/wiki), and
+serves the files locally:
+
+```bash
+bundle install
+bundle exec ruby build.rb --serve
```

Comment:
Will leave this as-is, as I believe the alternative is more confusing :+1:

Reply:
> I originally had `cd docs/` - but I was worried that will be confusing to folk that have already cd'd into this folder to view the readme. Especially as there will be a `docs` folder here as well after `build.rb` is run 😄

I presumed that people would know that the `build.rb` would have to be in the current directory given the 3rd instruction here but people might get confused and try running bundle install inside the main Metasploit directory which is why I suggested this. If you'd prefer to maybe add a note explaining this I think this would make sense but overall my goal was just to clear up this confusion.

Reply:
Added an explicit `cd` step

https://github.com/rapid7/metasploit-framework/pull/15923/commits/ca2f19cca38bfd4dbd87e696dc685bdd71a1638d

Reply:
Resolved by changing this to `cd metasploit-framework/docs`. Marking this as resolved.